### ISSUE TITLE
gallery jigs still showing "draft" after published - should remove ui indicator

### ISSUE THREAD

**Issue by dakom** [ji-devs/ji-cloud#2010]

**Comment by dakom** [comment]
@RickyLB reported this as resolved for now, so closing :)

**Comment by MendyBerger** [comment]
Don't think that this is a backend issue

**Comment by dakom** [comment]
Right- he had just reported it as resolved already from a user perspective when he tried it. I didn't check it myself... I only closed it because I had opened it and thought maybe I missed something. Of course only close when it's confirmed to be fixed :)
